openstack/standalone-cinder/pkg/provisioner/rbd.go

```diff
@@ -24,7 +24,9 @@ import (
 	"github.com/golang/glog"
 	"github.com/kubernetes-incubator/external-storage/lib/controller"
 	"github.com/kubernetes-incubator/external-storage/openstack/standalone-cinder/pkg/volumeservice"
+
 	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/api/v1/helper"
```

Comment:
hold till https://github.com/kubernetes-incubator/external-storage/pull/578 is resolved

Reply:
helper pkg is changed now, please rebase and update